curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-dom-manipulation-by-building-a-role-playing-game/62a8b1762b7775124622e1a3.md

```diff
@@ -1,32 +1,28 @@
 ---
 id: 62a8b1762b7775124622e1a3
-title: Step 60
+title: Step 59
 challengeType: 0
-dashedName: step-60
+dashedName: step-59
 ---
 
 # --description--
 
 Instead of assigning the `innerText` and `onclick` properties to specific strings and functions, the `update` function will use data from the `location` that is passed into it. First, that data needs to be passed.
```

Comment:
isn't it, `first the function needs to be classed` ? better remove it 

```suggestion
Instead of assigning the `innerText` and `onclick` properties to specific strings and functions, the `update` function will use data from the `location` that is passed into it.
```